Assets/VRTK/Scripts/Utilities/ObjectFollow/VRTK_ObjectFollow.cs

```diff
@@ -13,8 +13,15 @@ public abstract class VRTK_ObjectFollow : MonoBehaviour
         [Tooltip("The game object to change the property values of. If left empty the game object this script is attached to will be changed.")]
         public GameObject gameObjectToChange;
 
+        [Tooltip("Whether to follow the position. CAUTION: This value overwrite gameObjectToFollow position! Set to Vector3.zero for use gameObject method")]
+        public Vector3 positionToFollow = Vector3.zero;
+
         [Tooltip("Whether to follow the position of the given game object.")]
         public bool followsPosition = true;
+        public bool followPosX = true;
+        public bool followPosY = true;
+        public bool followPosZ = true;
```

Comment:
The various bools `followPos*` should probably be a bitfield enum instead.

Reply:
With an enum you should not exclude / include certain axes. Am I wrong?